test/Microsoft.ML.Tests/Transformers/ValueMappingTests.cs

```diff
@@ -89,6 +89,9 @@ public void ValueMapInputIsVectorTest()
             var estimator = new WordTokenizingEstimator(Env, new[]{
                     new WordTokenizingTransformer.ColumnInfo("TokenizeA", "A")
                 }).Append(new ValueMappingEstimator<ReadOnlyMemory<char>, int>(Env, keys, values, new[] { ("VecD", "TokenizeA"), ("E", "B"), ("F", "C") }));
+            var schema = estimator.GetOutputSchema(SchemaShape.Create(dataView.Schema));
+            Assert.True(schema.TryFindColumn("VecD", out var originalColumn));
+            Assert.Equal(SchemaShape.Column.VectorKind.VariableVector, originalColumn.Kind);
```

Comment:
>originalColumn [](start = 71, length = 14)

can u also add test for values for this vector.

i believe it should be <2, 3 1> ?

Reply:
If you look at the following code, it does exactly what meant.

---
In reply to: [253622276](https://github.com/dotnet/machinelearning/pull/2386#discussion_r253622276) [](ancestors = 253622276)